scripts/generate_report.sh

```
@@ -80,152 +115,177 @@ STATS=$(gh api graphql -f query='
-log "- Current day: $CURRENT_DAY"
-log "- Remaining days: $REMAINING_DAYS"
-
-# Slack通知用のJSONペイロードを作成
+###
+### 5. Slack通知ブロック作成
+###
 log "Creating Slack payload..."
+
+# オーガニゼーション毎のブロックを作成
+ORG_BLOCKS=()
+for org in $ORGS
```

Shellcheck (0.10.0)

[warning] Expanding an array without an index only gives the first element.

(SC2128)